packages/protocol-contracts/test/deterministicDeployment.utils.ts

```diff
@@ -0,0 +1,38 @@
+import { Provider, TransactionReceipt } from "@ethersproject/providers";
```

Comment:
I think I'd go for `deterministic-deployment` (kebab case instead of camelCase).